inf-clojure.el

```diff
@@ -250,6 +262,25 @@ one process, this does the right thing.  If you run multiple
 processes, you might need to change `inf-clojure-buffer' to
 whichever process buffer you want to use.")
 
+(defvar inf-clojure-mode-hook '()
```

Comment:
Seems you didn't merge something cleanly. This definition and the subsequent `put` were deleted recently.